projects/packages/sync/src/modules/class-woocommerce-products.php

```diff
@@ -18,12 +18,16 @@
  * Note: This module is currently used for analytics purposes only.
  */
 class WooCommerce_Products extends Module {
+
+	const PRODUCT_POST_TYPES = array( 'product', 'product_variation' );
```

Comment:
We could use this in https://github.com/Automattic/jetpack/blob/0848aeaa589cd5f5edbba1a1b5c555259047fe25/projects/packages/sync/src/modules/class-woocommerce-products.php#L408

Reply:
Thanks, updated in 206b04cebb971e46873e9f2ff686ba4aa861fd72 🙂